device-code-flow-console/Program.cs

```diff
@@ -53,5 +55,15 @@ static void Main(string[] args)
                 Console.ResetColor();
             }
         }
+
+        private static async Task RunAsync()
+        {
+            AuthenticationConfig config = AuthenticationConfig.ReadFromJsonFile("appsettings.json");
+            var app = new PublicClientApplication(config.ClientId, config.Authority);
+            var httpClient = new HttpClient();
```

Comment:
Do you have a reason to inject the httpClient into MyInformation ? I know it is good practice to use a single instance of HttpClient in the app, but MyInformation is the only component that uses it, so might as well let it create it.